Split tree import out to a separate step

ttung opened the pull request:
### Description
This is necessary because we need to import both the internal and the external samples *before* we import trees.  It is probably even a good idea to import _all_ samples before we import any trees.

Depends on #298 

#### Issue
[ch133318](https://app.clubhouse.io/genepi/story/133318/port-over-internal-samples-from-covidhub-db-into-appropriate-projects-about-500)

### Test plan
Ran through this sequence:
1. import all the marin users.
2. import all the marin samples (internal and external).
3. import all the marin trees.

shanaxel42 commented on src/backend/aspen/covidhub_import/import_trees.py:
```diff
@@ -0,0 +1,136 @@
+        dst_prefix_url = S3UrlParser(s3_dst_prefix)
+        for key in list_bucket(s3_src, src_prefix_url.bucket, src_prefix_url.key):
+            key_mo = re.match(
+                r".*_(?P<year>\d{2})(?P<month>\d{2})(?P<day>\d{2})\.json",
```
This is some serious regex

Commit:
Split tree import out to a separate step

### Description
This is necessary because we need to import both the internal and the external samples *before* we import trees.  It is probably even a good idea to import _all_ samples before we import any trees.

#### Issue
[ch133318](https://app.clubhouse.io/genepi/story/133318/port-over-internal-samples-from-covidhub-db-into-appropriate-projects-about-500)

### Test plan
Ran through this sequence:
1. import all the marin users.
2. import all the marin samples (internal and external).
3. import all the marin trees.
